src/cephadm/cephadm.py

```diff
@@ -5542,6 +5596,9 @@ def _get_parser():
         'disk-rescan', help='rescan all HBAs to detect new/removed devices')
     parser_disk_rescan.set_defaults(func=command_rescan_disks)
 
+    parser_list_images = subparsers.add_parser(
+        'list_images', help='list all the default images')
```

Comment:
By convention cephadm uses dashes between terms in subcommands names, so `list-images` not `list_images`.